sagemaker-pipelines/tabular/lambda-step/sagemaker-pipelines-lambda-step.ipynb

```diff
@@ -0,0 +1,903 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #22.        lambda_func=Lambda(
</pre><p>I think we should take the creation of lambda object out of lambda step configuration so that the object can be reused for manipulating lambdas. For example we create that object separately and then while configuring LambdaStep, just say <code>lambda_function=func</code></p><p>This <code>func</code>  object then could be used in cleanup below to delete that lambda function directly by just calling <code>func.delete()</code></p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2845/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
Fixed

Reply:
<p>missed to fix this?</p>